tests/draft2019-09/optional/format/ipv6.json

```diff
@@ -112,6 +112,41 @@
                 "description": "ipv4 segment must have 4 octets",
                 "data": "1:2:3:4:1.2.3",
                 "valid": false
+            },
+            {
+                "description": "leading whitespace is invalid",
+                "data": "  ::1",
+                "valid": false
+            },
+            {
+                "description": "trailing whitespace is invalid",
+                "data": "::1  ",
+                "valid": false
+            },
+            {
+                "description": "netmask is not a part of ipv6 address",
+                "data": "fe80::/64",
+                "valid": false
+            },
+            {
+                "description": "zone id is not a part of ipv6 address",
+                "data": "fe80::a%eth1",
```

Comment:
Oy, looks like `inet_pton` happily accepts this, which is how I assume most folks have actually implemented IPv6 validation...